packages/trip-form/src/MetroModeSelector/index.tsx

```diff
@@ -220,38 +186,18 @@ function ModeButton({
     },
     [onPopupClose, setOpen]
   );
-  const {
-    context,
-    floating,
-    middlewareData: { arrow: { x: arrowX, y: arrowY } = {} },
-    reference,
-    strategy,
-    x,
-    y
-  } = useFloating({
+  const { context, reference } = useFloating({
     middleware: [offset(8), shift(), arrow({ element: arrowRef })],
     onOpenChange,
     open
   });
 
-  const { getFloatingProps, getReferenceProps } = useInteractions([
-    useHover(context, {
-      // Enable hover only if no popup has been triggered via keyboard.
-      // (This is to avoid focus being stolen by hovering out of another button.)
-      enabled: itemWithKeyboard === null && hoverEnabled,
-      handleClose: safePolygon({
-        blockPointerEvents: false,
-        restMs: 500,
-        buffer: 0
-      })
-    }),
+  const { getReferenceProps } = useInteractions([
```

Comment:
I think you can remove ALL of this! We don't need the floating UI stuff at all anymore since it was only being used for this. I think even the dependency can go!